ceph_deploy/mds.py

```diff
@@ -122,6 +122,8 @@ def create_mds(distro, name, cluster, init):
     if distro.is_el:
         system.enable_service(distro.conn)
 
+    if distro.distro == 'suse':
+        system.enable_service(distro.conn, 'ceph')
```

Comment:
Since ``system.enable_service(distro.conn)`` and ``system.enable_service(distro.conn, 'ceph')`` do the same thing, and since the only different between ``distro.is_el`` and ``distro.is_rpm`` is adding SUSE to ``is_el`` (https://github.com/ceph/ceph-deploy/blob/master/ceph_deploy/hosts/__init__.py#L63-L65), can we instead implement this by just chaning ``distro.is_el`` to ``distro.is_rpm``?

I don't think we need an entirely new if case to handle this.

This goes for both this change and the one in osd.py.